submissions/nwokejeme-immaculeta/dom/signup.js

```diff
@@ -0,0 +1,53 @@
+let submitForm = document.getElementById('validateForm'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    otherName = document.getElementById('otherName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    email = document.getElementById('email');
+    password = document.getElementById('password');
+    confirmPassword = document.getElementById('confirmPassword');
+
+let validate = (field, event, regEx, result, output) => {
+    if (field.value === "") {
+        field.nextElementSibling.innerHTML = result;
+        event.preventDefault();
+        return false;
+    } else if (!field.value.match(regEx)) {
+        field.nextElementSibling.innerHTML = output;
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+    }
+}
+
+let passwordValidation = (
```

Comment:
Try to break the line of code down. And is this a function? If yes, why is it not invoke and it is not properly written. The only parameter it should hold is `event`. And if no, then it is not supposed to be so and I don't understand this kind of variable declaration